aion/web/views/user_view.ex

```diff
@@ -0,0 +1,17 @@
+defmodule Aion.UserView do
+  use Aion.Web, :view
+
+  def render("user.json", %{user: user}) do
+    if user != nil do
```

Comment:
I'm still not sure about conventional way of dealing with nils, but [this answer](https://stackoverflow.com/a/37781248) and [Jose's answer](https://groups.google.com/d/msg/elixir-lang-talk/T6fg8h60Nnw/PodQeBGzGAAJ) suggest some solutions. I think the second one could actually be better (but it's just my personal opinion because I'm not a die hard fan of multiple function clauses). Here's how the first one could work: 
```
def render("user.json", %user: user}) when is_nil user, do: %{}
def render("user.json", %user: user}), do: Map.fetch(user, [:id, :name, :email])
```
Similarly to my suggestion ^ in the second one you should definitely use `Map.fetch/2`

Reply:
Would this work?
```ex
def render("user.json", %{user: nil}), do: %{}
def render("user.json", %{user: user}), do: Map.fetch(user, [:id, :name, :email])
```

Reply:
`Map.take/2`*

Reply:
yeah right, was looking at the docs and eventually used the wrong one
lgtm